app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,14 @@
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+div#intro {
+    margin-top: 100px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-left, margin-top, width